fem/qspace.cpp

```diff
@@ -129,7 +129,7 @@ void FaceQuadratureSpace::ConstructOffsets()
       }
       face_indices[f_idx] = i;
       offsets[f_idx] = offset;
-      Geometry::Type geom = mesh.GetFaceGeometry(i);
+      Geometry::Type geom = mesh.GetFaceGeometryType(i);
```

Comment:
So, one method works with ghost faces and not the other one, otherwise the two methods do the same thing (?!). I guess we should add a debug check in `GetFaceGeometry` like `MFEM_ASSERT( Face < NumOfFaces, "This method does not handle ghost faces, use 'GetFaceGeometryType' instead.");` or deprecate `GetFaceGeometry`, or make `GetFaceGeometry` call `GetFaceGeometryType`.

Reply:
My thoughts exactly 😄 https://github.com/mfem/mfem/pull/3320#issuecomment-1311979056